File: BeerPlease/app/src/main/java/app/dgandroid/eu/beerplease/controllers/Manager.java
package app.dgandroid.eu.beerplease.controllers;

/**
 * Created by Duilio on 23/05/2017.
 */

public class Manager {

    private static Manager instance;
    private int page = 1;

    public static Manager getInstance(){
        if(instance == null) {
            instance = new Manager();
        }
        return instance;
    }

    public void incrementPage(){
        this.page++;
    }

    public void clear(){
        this.page = 1;
    }

    public int getPage(){
        return this.page;
    }
}

File: BeerPlease/app/src/main/java/app/dgandroid/eu/beerplease/controllers/Manager.kt
package app.dgandroid.eu.beerplease.controllers

/**
 * Created by Duilio on 23/05/2017.
 */

object Manager {
    var page = 1
        private set

    fun incrementPage() {
        this.page++
    }

    fun clear() {
        this.page = 1
    }
}